chat-widget/src/components/livechatwidget/livechatwidgetstateful/LiveChatWidgetStateful.tsx

```diff
@@ -158,26 +160,51 @@ export const LiveChatWidgetStateful = (props: ILiveChatWidgetProps) => {
             }
         });
 
-        // start chat from SDK Event
+        // Start chat from SDK Event
         BroadcastService.getMessageByEventName(BroadcastEvent.StartChat).subscribe(() => {
             TelemetryHelper.logActionEvent(LogLevel.INFO, {
                 Event: TelemetryEvent.StartChatEventRecevied,
                 Description: "Start chat event received."
             });
-            if (state.appStates.isMinimized) {
-                dispatch({ type: LiveChatWidgetActionType.SET_MINIMIZED, payload: false });
-            } else {
+            // Getting updated state from cache
+            const widgetStateEventName = getWidgetCacheId(chatSDK?.omnichannelConfig?.orgId ?? "", chatSDK?.omnichannelConfig?.widgetId ?? "");
+            const widgetStateFromCache = DataStoreManager.clientDataStore?.getData(widgetStateEventName, "localStorage");
+            const persistedState = widgetStateFromCache ? JSON.parse(widgetStateFromCache) : undefined;
+
+            if (persistedState.appStates.conversationState === ConversationState.Closed ||
+                persistedState.appStates.conversationState === ConversationState.InActive ||
+                persistedState.appStates.conversationState === ConversationState.Postchat) {
+                BroadcastService.postMessage({
+                    eventName: BroadcastEvent.ChatInitiated
+                });
                 prepareStartChat(props, chatSDK, state, dispatch, setAdapter);
+            } else {
+                dispatch({ type: LiveChatWidgetActionType.SET_MINIMIZED, payload: false });
+                BroadcastService.postMessage({
+                    eventName: BroadcastEvent.MaximizeChat,
```

Comment:
We don't need to differentiate maximizeChat scenarios between new chat and minimized chat, as discussed

Reply:
This is used for calling Maximize chat on calling `startChat` on a minimized chat.